exporter/general_collector_test.go

```diff
@@ -37,15 +37,22 @@ func TestGeneralCollector(t *testing.T) {
 	client := tu.DefaultTestClient(ctx, t)
 	c := newGeneralCollector(ctx, client, logrus.New())
 
+	filter := []string{
+		"collector_scrape_time_ms",
+	}
+	count := testutil.CollectAndCount(c, filter...)
```

Comment:
Metric value may fluctuate for every run.  Hence checking only presence of metric.